.github/workflows/develop.yml

```diff
@@ -19,6 +19,8 @@ jobs:
 
       - name: Setup node
         uses: actions/setup-node@v2
+        with:
+          node-version: '16'
```

Comment:
feels weird we weren't specifying a version here before, I'm wondering what version we were getting?